app/create_app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, render_template, request, redirect
```

Comment:
You've created a new file app/create_app.py but also kept todo_app/app.py etc.
Delete the original, out of date files!

That kind of thing is bound to cause confusion